base_custom_info/static/src/scss/custom_info.scss

```diff
@@ -0,0 +1,3 @@
+.o_form_view .o_field_widget .o_field_widget, .o_form_view .btn .o_field_widget .custom_info_value {
+        margin-bottom: 5px;
```

Comment:
Is this really used or required? Can you replace it with some class on the widget such as `mb-5`?

Reply:
I was unable due to:
https://github.com/odoo/odoo/blob/12.0/addons/web/static/src/scss/form_view.scss#L394
I think now the code is clearer.